game-core/src/main/java/games/strategy/triplea/player/AbstractBasePlayer.java

```diff
@@ -43,38 +50,6 @@ public GameData getGameData() {
     return playerBridge.getGameData();
   }
 
-  /**
-   * Get the IPlayerBridge for this game player.
-   * (This is not a delegate bridge, and we cannot send changes on this. Changes should only be done within a delegate,
-   * never through a
-   * player.)
-   */
-  protected final IPlayerBridge getPlayerBridge() {
-    return playerBridge;
-  }
-
-  @Override
-  public final String getName() {
-    return name;
-  }
-
-  @Override
-  public final String getType() {
-    return type;
-  }
-
-  @Override
-  public final PlayerID getPlayerId() {
-    return playerId;
-  }
-
-  @Override
-  public String toString() {
-    return (playerId == null || playerId.getName() == null || !playerId.getName().equals(name))
-        ? (type + ":" + name + ":" + (playerId == null ? "NullID" : playerId.getName()))
-        : (type + ":" + name);
-  }
```

Comment:
Was the string returned by this method used for display in the UI someplace?  It's just that the formatting looks suspicious, and changing it may have unintended consequences.

Reply:
`PlayerBridge` is pretty internal, very unlikely to be displayed. Also a tip off "NullID"